src/redux/coronaData/coronaData.actions.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Same here.  Let's not disable linter rules. These are here to help us write clean code.